connectors_sdk.gemspec

```diff
@@ -27,4 +27,21 @@ Gem::Specification.new do |s|
   s.add_dependency 'activesupport'
   s.add_dependency 'bson'
   s.add_dependency 'mime-types'
+  s.add_dependency 'tzinfo-data'
+  s.add_dependency 'nokogiri'
+
+  # Dependencies for the HTTP service
+  s.add_dependency 'sinatra'
```

Comment:
Does it make sense that the _gem_ relies on Sinatra (and the other deps here)? We shouldn't actually be including any of the web app code in the gem.

Reply:
You're probably right. After 3 in a row of, "ugh, we use that too?" I just added everything to unblock myself. But I'll pair this list back down.

Reply:
updated, I was able to remove a few, but not many.

Reply:
I could get rid of signet if we pulled all the `authorization.rb` files into the `connectors_app/` dir. But IDK if we want to go through so much restructuring.